Error while installing in Windows

Facing the following error when trying to install in Windows.

C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(209
,5): error MSB6006: "cmd.exe" exited with code 1. [C:\aerospike-vm\aerospikedemo\node_modules\aerospike\build\aerospike
-client-c.vcxproj]
gyp ERR! build error
gyp ERR! stack Error: `C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\MSBuild\15.0\Bin\MSBuild.exe` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onExit (C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\lib\build.js:262:23)
gyp ERR! stack     at ChildProcess.emit (events.js:198:13)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:248:12)
gyp ERR! System Windows_NT 10.0.18362
gyp ERR! command "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild"
gyp ERR! cwd C:\aerospike-vm\aerospikedemo\node_modules\aerospike
gyp ERR! node -v v10.16.0
gyp ERR! node-gyp -v v3.8.0
gyp ERR! not ok
npm WARN aerospikedemo@1.0.0 No repository field.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! aerospike@3.13.0 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the aerospike@3.13.0 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:

I’ll need more info than this. Esp. since the [Windows CI builds in AppVeyor](https://ci.appveyor.com/project/aerospike/aerospike-client-nodejs) don’t show any issues.

Can you provide the full log that should have been listed after this line:

> npm ERR! A complete log of this run can be found in:

If it’s too large to post here, maybe you can post it as a Gist on GitHub or on pastebin.com?

Same here. Several developers trying in their Win machine and facing the same issue as below.
First time installation for all of us.

I don't see *.msbuild.log file. Where can I see the log?

 Installing Aerospike C Client dependency
  ..\scripts\build-c-client.ps1 : File
  C:\Users\Santha\AppData\Roaming\npm\node_modules\aerospike\scripts\build-c-client.ps1 cannot be loaded because runnin
  g
  scripts is disabled on this system. For more information, see about_Execution_Policies at
  https:/go.microsoft.com/fwlink/?LinkID=135170.
  At line:1 char:1
  + ..\scripts\build-c-client.ps1 -Configuration Release -NodeLibFile C:\ ...
  + ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      + CategoryInfo          : SecurityError: (:) [], PSSecurityException
      + FullyQualifiedErrorId : UnauthorizedAccess
C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(209
,5): error MSB6006: "cmd.exe" exited with code 1. [C:\Users\Santha\AppData\Roaming\npm\node_modules\aerospike\build\aer
ospike-client-c.vcxproj]
gyp ERR! build error
gyp ERR! stack Error: `C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\MSBuild\15.0\Bin\MSBuild.exe` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onExit (C:\Users\Santha\AppData\Roaming\npm\node_modules\npm\node_modules\node-gyp\lib\build.js:194:23)
gyp ERR! stack     at ChildProcess.emit (events.js:198:13)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:248:12)
gyp ERR! System Windows_NT 10.0.18362
gyp ERR! command "C:\\Program Files\\nodejs\\node.exe" "C:\\Users\\Santha\\AppData\\Roaming\\npm\\node_modules\\npm\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild"
gyp ERR! cwd C:\Users\Santha\AppData\Roaming\npm\node_modules\aerospike
gyp ERR! node -v v10.16.0
gyp ERR! node-gyp -v v5.0.5
gyp ERR! not ok
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! aerospike@3.13.0 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the aerospike@3.13.0 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Santha\AppData\Roaming\npm-cache\_logs\2019-10-31T17_49_33_671Z-debug.log

--- a/README_WINDOWS.md
+++ b/README_WINDOWS.md
@@ -10,13 +10,16 @@ and suitable for application prototyping and development.
 ## Prerequisites
 
 * Windows 7 or later
-* Visual C++ 2015 Build Tools or later
-* Node.js v6.x (LTS) or later
+* Windows 8.1 SDK
+* Visual C++ 2015 Build Tools
+* Node.js v8.x (LTS) or later
 
 The package includes a native add-on. To compile the add-on, Microsoft's Visual
 C++ Build Tools 2015 are required. The easiest way to install the build tools,
-is to install the [`windows-build-tools`
-](https://www.npmjs.com/package/windows-build-tools) package using npm.
+is using the the [`windows-build-tools`
+](https://www.npmjs.com/package/windows-build-tools) npm package:
+
+    npm install windows-build-tools --global --vs2015
 
 The Aerospike Node.js client depends on the Aerospike C client. During
 installation, a copy of the C client SDK is downloaded and compiled.
@@ -26,3 +29,14 @@ installed via the
 nuget package. Please refer to the [Aerospike C client
 documentation](https://github.com/aerospike/aerospike-client-c/tree/master/vs)
 for further information.
+
+<a name="FAQ"></a>
+## Frequently Asked Questions
+
+#### While installing the client, I am getting an error that "build-c-client.ps1 cannot be loaded because running scripts is disabled on this system". What do I need to do?
+
+PowerShell's execution policy prevents you from running the client's installation script. You can temporarily lift the restrictions for the current PowerShell session by running the following command:
+
+    Set-ExecutionPolicy -ExecutionPolicy Unrestricted -Scope Process
+    
+For further information, please refer to the PowerShell documentation [About Execution Policies](https://docs.microsoft.com/en-sg/powershell/module/microsoft.powershell.core/about/about_execution_policies).

--- a/README_WINDOWS.md
+++ b/README_WINDOWS.md
@@ -16,7 +16,7 @@ and suitable for application prototyping and development.
 
 The package includes a native add-on. To compile the add-on, Microsoft's Visual
 C++ Build Tools 2015 are required. The easiest way to install the build tools,
-is using the the [`windows-build-tools`
+is using the [`windows-build-tools`
 ](https://www.npmjs.com/package/windows-build-tools) npm package:
 
     npm install windows-build-tools --global --vs2015

